CHANGELOG.md

```diff
@@ -6,7 +6,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 ### Changed
-- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0)
+- Upgraded `jackson-databind` to 2.9.7 (was 2.9.0).
+- Upgraded `hive-jdbc` to 2.3.4 (was 2.0.0).
+- Upgraded `hotels-oss-parent` to 4.0.1 (was 2.0.6).
+- Changed dependency from `com.microsoft.sqlserver.sqljdbc` (4.1) to `com.microsoft.sqlserver.sqljdbc4` (4.0).
```

Comment:
This was the main change I did to get it building locally, the other changes are nice to haves but put us in a better position for building further on this.

Reply:
Yeah makes sense